docs/_layouts/documentation.html

```diff
@@ -27,5 +27,6 @@
   </div>
 </div>
 
+<style>body{overflow: hidden;}</style>
```

Comment:
The guilty div is the body.This only occurs on certain browser (for me in FF with the scrollbars always activated). It's because we use inner scrollable panels and only on the documentation...

It's inline and it's ugly but a correct solution would be to name the body with a class and use a specific layout.